include/adt/edgeset.h

```diff
@@ -17,7 +17,7 @@ struct edge_set;
 struct state_set;
 
 struct edge_iter {
-	size_t i;
+	size_t i, j;
```

Comment:
These names are vague because the struct is currently shared between the two implementations, which use the counters differently. The iterator isn't opaque because most of the callers stack-allocate it.